recipes/cctk/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
you can include this one-liner in the meta.yaml if you want

Reply:
Thanks! I'll move it.